src/lib/keypom.ts

```diff
@@ -191,6 +198,75 @@ class KeypomJS {
   getDropMetadata = (metadata: string) =>
     JSON.parse(metadata || JSON.stringify({ dropName: 'Untitled' }));
 
+  deleteDrops = async ({ wallet, dropIds }) => await deleteDrops({ wallet, dropIds });
+
+  deleteKeys = async ({ wallet, dropId, publicKeys }) =>
+    await deleteKeys({ wallet, dropId, publicKeys });
+
+  getDropInfo = async (dropId?: string): Promise<ProtocolReturnedDrop> => {
+    if (!dropId) throw new Error('Missing dropId');
+    // TODO: add basic catch
+    return await getDropInformation({ dropId });
```

Comment:
```suggestion
    return getDropInformation({ dropId });
```

Reply:
some fun fact [here](https://dmitripavlutin.com/return-await-promise-javascript/)

if u use 
```
try { return await Promise.reject(new Error()) }
catch (err) { // error is caught }
```
if u use
```
try { return Promise.reject(new Error()) }
catch (err) { // error is not caught }
}
```
very interesting

Reply:
with that logic, shouldnt it keep the `await` though, since `getDropInformation` returns a promise. If we only do a `return getDropInformation({ dropId });`, the function will not wait for the return value and will return missing stack error instead.

Reply:
although, i agree to standardize the catch to `try catch`

Reply:
after discussion will follow eric's